keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,1022 @@
+  - Should we just do this all of the time post-release?
+- (to Josh) What did we discover regarding feature trajectory?
+
+Thoughts:
+I would want requesters to be very explicit about the kind of data we're interested in.
+SIG Release and others can work on collection, but we need to make sure this isn't a continually moving target.
+
+We're also starting from a disadvantage trying to compare our status quo to something we haven't tried for a sustained period of time.
+
+#### TODO Blocking upgrade tests
+
+Aaron C:
+
+> Can we make upgrade jobs / tests blocking to make the upgrade between versions better
+
+#### TODO More automation
+
+@vincepri:
+
+> There is not enough automation (true for probably all our projects and repositories).
+
+#### TODO Improve visibility
+
+@vincepri:
+
+> Changes are hard to keep up with, and sometimes important things are buried in release notes.
+
+#### TODO More policy
+
+@vincepri:
+
+> Some fear that less releases without policies (read: saying "no" more) isn't enough.
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+#### TODO Release Team
+
+@saschagrunert:
+
+> On the other hand it will give less people a chance to participate in the shadowing program for example. Anyways, I think we will find appropriate solutions around those kind of new challenges.
+
+@jeremyrickard:
+
+> The one downside is that we will remove an opportunity for shadowing, and as we saw this time around we had >100 people apply, and this will remove ~24-ish opportunities. I think we can maybe identify some opportunities for folks that want to be involved though. takes off release lead hat
+
+@wilsonehusin:
+
+> as someone who started getting involved in this project through shadowing release team, I'd like to echo what @saschagrunert & @jeremyrickard raised above regarding shadow opportunities -- I'm glad we're acknowledging the downside and hope we can keep in mind to present other opportunities for folks to get involved!
+
+@kcmartin:
+
+> As to the potential for limiting shadow opportunities (mentioned by @jeremyrickard, @wilsonehusin, and others), I'm definitely tuned in to that being a downside, since I've served as a SIG-Release shadow three times, and I think it's a fantastic opportunity!
+>
+> One possible way to alleviate that downside would be to have 5 shadows, instead of three or four, per sub-team. I believe this is still a manageable number for the Leads, and could distribute the work more evenly.
+
+@pires:
+
+> On a more personal note, (@jeremyrickard wink, wink) I applied for release shadow believing I'd be picked given my past contributions to the project and my justification to be selected over others. Being rejected was a humbling experience and I'm happy to let you know I didn't lose any of the appetite to contribute. Others may feel differently but, then again, the project is maturing and so should the community.
```

Comment:
Depending on how many shadows we select per role, we could state that we now select more shadows with a cadence of 3 releases per year.

Reply:
Marked as out-of-scope in 8d84fcbdb208b27db9187edec05d56937e048bec.
Separate issue to discuss opened here: https://github.com/kubernetes/sig-release/issues/1494